support/cas-server-support-person-directory/src/main/java/org/apereo/cas/config/CasPersonDirectoryConfiguration.java

```diff
@@ -260,11 +264,31 @@ public void configureAttributeRepositoryPlan(final PersonDirectoryAttributeRepos
     @Configuration("CasPersonDirectoryLdapConfiguration")
     public class CasPersonDirectoryLdapConfiguration implements PersonDirectoryAttributeRepositoryPlanConfigurer {
 
+        @Bean
+        @RefreshScope
```

Comment:
Necessary?

Reply:
`ldapAttributeRepositories` is a `RefreshScope`. If `ldapAttributeRepositories` refreshes then the corresponding `ConnectionFactory` should be release too to save resources.

Reply:
But anyway this is no longer relevant now after refactoring.